dev/smoketest/main.tf

```diff
@@ -46,7 +46,8 @@ resource "kubernetes_pod" "smoketest" {
 
     labels = {
       "allow-to-bitcoind" : "true",
-      "allow-to-lnd" : "true"
+      "allow-to-lnd" : "true",
+      "allow-to-fulcrum" : "true"
```

Comment:
I wonder if this is required

Reply:
https://github.com/GaloyMoney/charts/pull/2428 if this passes then we can remove this change

Reply:
removed in https://github.com/GaloyMoney/charts/pull/2418/commits/65ece7ebcbb5d91336b280cada54b8101d17dfab